pyproject.toml

```diff
@@ -25,10 +25,24 @@ ignore_missing_imports = true
 follow_imports = 'silent'
 warn_redundant_casts = true
 show_error_codes = true
+enable_error_code = [
+    'ignore-without-code'
+    ]
+# The following list of codes are globally ignored, do not add to this list
+disable_error_code = [
+    # annotation-unchecked are 'warning notes', not errors and won't cause mypy to fail
+    # but it does create a lot of noise in the CI mypy step.
+    # https://mypy-lang.blogspot.com/2022/11/mypy-0990-released.html
+    'annotation-unchecked'
+    ]
```

Comment:
Just want to call out that I am globally disabling this "error" code, which we should generally never do.

But in this case, this "error" code doesn't actually result in an error anyways, so it wouldn't actually prevent any bugs without an additional configuration setting like [`check-untyped-defs`](https://mypy.readthedocs.io/en/stable/config_file.html?highlight=check-untyped-defs#confval-check_untyped_defs)